Core/clim-core/table-formatting.lisp

```diff
@@ -155,24 +160,28 @@ to a table cell within the row."))
                                (row-record standard-row-output-record))
   (map-over-block-cells function row-record))
 
+(defgeneric invoke-formatting-row (stream continuation
+                                   &rest initargs
+                                   &key record-type &allow-other-keys)
+  (:method (stream continuation &rest initargs
+                                &key (record-type 'standard-row-output-record))
+    (apply #'invoke-with-new-output-record stream
+           (lambda (stream record)
+             (declare (ignore record))
+             (funcall continuation stream))
+           record-type (alexandria:remove-from-plist initargs :record-type))))
```

Comment:
with-keyword-removed

Reply:
Instead of `remove-from-plist`?

Reply:
yes, closing (because this remark is in the new review as well)